docs/content/preview/yugabyte-platform/anywhere-automation/anywhere-cli.md

```diff
@@ -0,0 +1,273 @@
+---
+title: yba CLI
+headerTitle: YugabyteDB Anywhere CLI
+linkTitle: yba CLI
+description: Use YugabyteDB Anywhere CLI to create and manage universes.
+headcontent: Install yba, configure default settings, and set up autocompletion
+earlyAccess: /preview/releases/versioning/#feature-availability
+menu:
+  preview_yugabyte-platform:
+    parent: anywhere-automation
+    identifier: anywhere-cli
+    weight: 50
+type: docs
+rightNav:
+  hideH4: true
+---
+
+The [YugabyteDB Anywhere (YBA) Command Line Interface](https://github.com/yugabyte/yugabyte-db/tree/master/managed/yba-cli) (yba) is an open source tool that enables you to interact with YBA accounts using commands in your command-line shell. With minimal configuration, you can start running commands from the command line that implement functionality equivalent to that provided by the browser-based YBA interface.
+
+## Install yba
+
+To install yba, download the appropriate binary from the yba-cli GitHub project [releases](https://github.com/yugabyte/yba-cli/releases).
+
+On MacOS and Linux, you can also install yba using [Homebrew](https://brew.sh) by entering the following command:
+
+```sh
+brew install yugabyte/tap/yba
+```
```

Comment:
There is no brew install support yet. So need to remove this section too.

We can add this install instruction instead -

The YugabyteDB Anywhere installation already comes packaged with the CLI binary. This can be found in your YBA under the filesystem location `/opt/yugabyte/yugaware/yba-cli`. There are four flavours of the CLI binary.
```
yba_cli-<ver>-darwin-amd64
yba_cli-<ver>8-darwin-arm64
yba_cli-<ver>-linux-amd64 
yba_cli-<ver>-linux-arm64
```
Copy the appropriate binary from this location to your local machine from where you want to run the CLI. Note that the CLI can be used to interact only with a same or older version of YBA.